### DISCUSSION TITLE
Training Modpack Has Panicked! Vs. 1733 Yuzu (Also tried the latest vs of Ryujinx), Smash Ultimate 13.0.1, latest stable modpack release

### DISCUSSION THREAD

**Discussion by BroAkroze** [jugeeya/UltimateTrainingModpack#681] [Q&A] [answered]
![image_2024-04-07_140057707](https://github.com/jugeeya/UltimateTrainingModpack/assets/96663410/3d83159c-41a9-4bcb-b83b-4a0e4477c479)

I know virtually nothing about debugging and what causes these file conflictions and such, so I apologize in advance for my inexperience. I have not been able to use my training modpack for several months on either Yuzu or Ryujinx. I believe problems started to arise when I updated my yuzu build, though I'm not entirely sure. What I do know is that I did something, and afterwards my training modpack would either crash or do something incredibly weird with the textures, and the next day the game refused to start. This error above did NOT occur initially, rather the game would just claim to be loading but never actually start. 

As it turned out, all of my mods stopped working, so to troubleshoot I updated all of them and it worked. All except the training modpack, upon which it would give me this error. I need the training modpack in order to record for my videos, so I did about as much troubleshooting as I could given my limited knowledge. Here are all of the things I tried:

- Using Ryujinx. I used the latest stable release of the modpack, and an older version of Ryujinx (though I forget the exact version). Same error
- Using an older version of the modpack (not sure which version)
- Updating Ryujinx to the latest release
- Updating Atmosphere
- Using OpenGL in Yuzu (If I remember correctly it failed to load in a completely different way)
- Disabling and re-enabling the plugins
- Trying to find where the error is in the logs
- Disabling and re-enabling all other mods (save for the hooks necessary for it to run)
- Replacing the hooks I individually placed in there with the hooks bundled with the latest stable release

I do have a crash dump file I will include, though I don't necessarily know what I'm doing and what's in there may not be useful. 

What else can I try to get my training modpack working again? Thanks in advance to any responses.

For clarification the dump file below is from the day I began to troubleshoot this again (today), since until Yuzu's servers shut down I was okay with not having the training modpack.

[910d22bf-5fc6-44e0-91aa-bd04920824c6.dmp](https://github.com/jugeeya/UltimateTrainingModpack/files/14898404/910d22bf-5fc6-44e0-91aa-bd04920824c6.dmp)

**Comment by BroAkroze** [comment, marked as answer]
Using version 6.0.1 of the modpack has worked for me, at least for now. I'm sorry; I'm new to Github and put this in the wrong place. I saw another issue under the issues section denoting exactly what mine was, and was able to solve it from there.

**Reply by KeyBladeMaestro** [reply]
Hey @BroAkroze I was wondering if you knew where to find that specific ver of the Training mod pack?
There's a link to a discord that will help set up everything.
But when I follow the link to discord & launch my app nothing loads.

**Reply by BroAkroze** [reply]
I forgot exactly how I found it, but if you want me to guide you through it my Discord is BroAkroze#1826. I’ll take a look again and see where I found it, though I believe GitHub keeps track of past mod pack releases
